company-kata-solutions/src/test/java/org/eclipse/collections/companykata/Exercise8Test.java

```diff
@@ -94,7 +97,7 @@ public void totalOrderValuesByItem()
     {
         Function0<Double> zeroValueFactory = () -> 0.0;
         Function2<Double, LineItem, Double> aggregator = (result, lineItem) -> result + lineItem.getValue();
-        MutableMap<String, Double> map = null;
+        MutableMap<String, Double> map = this.company.getOrders().flatCollect(Order::getLineItems).aggregateBy(LineItem::getName, zeroValueFactory, aggregator);
```

Comment:
Requesting the same formatting change here as well.

Reply:
Done